source/_integrations/zoneminder.markdown

```diff
@@ -25,64 +26,12 @@ There is currently support for the following device types within Home Assistant:
 
 ## Configuration
 
-```yaml
-# Example configuration.yaml entry
-zoneminder:
-  - host: ZM_HOST
-```
-
-{% configuration %}
-host:
-  description: Your ZoneMinder server's host (and optional port), not including the scheme.
-  required: true
-  type: string
-path:
-  description: Path to your ZoneMinder install.
-  required: false
-  type: string
-  default: "`/zm/`"
-path_zms:
-  description: Path to the CGI script for streaming. This should match `PATH_ZMS` in ZM's "Paths" settings.
-  required: false
-  type: string
-  default: "`/zm/cgi-bin/nph-zms`"
-ssl:
-  description: Set to `true` if your ZoneMinder installation is using SSL.
-  required: false
-  type: boolean
-  default: false
-verify_ssl:
-  description: Verify the certification of the endpoint.
-  required: false
-  type: boolean
-  default: true
-username:
-  description: Your ZoneMinder username.
-  required: false
-  type: string
-password:
-  description: Your ZoneMinder password. Required if `OPT_USE_AUTH` is enabled in ZM.
-  required: false
-  type: string
-{% endconfiguration %}
-
-### Full configuration
-
-```yaml
-# Example configuration.yaml entry
-zoneminder:
-  - host: ZM_HOST
-    path: ZM_PATH
-    path_zms: ZM_PATH_ZMS
-    ssl: true
-    verify_ssl: true
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-```
+1. From the Home Assistant front-end, navigate to 'Configuration' then 'Integrations'. Under 'Set up a new integration' locate 'ZoneMinder' and click 'Configure'.
```

Comment:
I think we should keep some of the description of what to fill in the Config Flow. For example `path_zms` is `Path to the CGI script for streaming. This should match `PATH_ZMS` in ZM's "Paths" settings.`